ibericode/mailchimp-for-wordpress
# AMP compatibility

This PR adds AMP-compatibility to the forms. This is accomplished by adding the necessary response templates to forms and a new API endpoint that processes a form and returns its status.

If this needs `@since` docblock, you don't like where I've put the code, or any of the composer stuff needs to be removed, let me know.

Closes #457 

## To test
1. Install and activate [the official AMP plugin](https://wordpress.org/plugins/amp/). In AMP > General > Website Mode settings, set the mode to either Transitional or Standard.
2. Create a form and add it to a page with the mc4wp shortcode.
<img width="888" alt="Screen Shot 2020-01-22 at 11 34 30 AM" src="https://user-images.githubusercontent.com/7317227/72927557-551b2b80-3d0b-11ea-948c-0af6c0ffe71a.png">
3. In AMP and non-AMP mode try out the form. You should get the same look and behavior in each.
<img width="780" alt="Screen Shot 2020-01-22 at 11 35 15 AM" src="https://user-images.githubusercontent.com/7317227/72927550-52203b00-3d0b-11ea-9f55-383730019567.png">
